.github/workflows/api-binary-tests.yml

```diff
@@ -0,0 +1,133 @@
+
+
+name: Build and Test Binary Artifacts
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "api/source/**"
+      - "test/api/**"
+      - ".github/workflows/api-binary-tests.yml"
+  push:
+    branches:
+      - main
+    paths:
+      - "api/source/**"
+      - "test/api/**"
+      - ".github/workflows/api-binary-tests.yml"
+env:
+  STIGMAN_API_PORT: 64001
+  STIGMAN_DB_HOST: localhost
+  STIGMAN_DB_PORT: 3306
+  STIGMAN_DB_PASSWORD: stigman
+  STIGMAN_API_AUTHORITY: http://127.0.0.1:8080/auth/realms/stigman
+  STIGMAN_SWAGGER_ENABLED: true
+  STIGMAN_SWAGGER_SERVER: http://127.0.0.1:64001/api
+  STIGMAN_SWAGGER_REDIRECT: http://127.0.0.1:64001/api-docs/oauth2-redirect.html
+  STIGMAN_DEV_RESPONSE_VALIDATION: logOnly
+  STIGMAN_EXPERIMENTAL_APPDATA: 'true'
+  STIGMAN_OIDC_PROVIDER: http://127.0.0.1:8080/auth/realms/stigman
+jobs:
+  build-binary-artifacts:
+    name: Build binary artifacts
```

Comment:
Call existing binary build workflow? (it may need to be changed to allow workflow_call)